### PR TITLE
OpenCDC changes, part 1

### LINKED ISSUE #114: Implement OpenCDC changes
### Feature description

As the title implies.

### PR DESCRIPTION
**Pull request by hariso**
Fixes part 1 of #114.

In this PR, we keep the same level of functionality as before, but we use the updated API, with the OpenCDC changes. Since Kafka Connect connectors generally do not support CDC, this PR interprets all incoming records as created records. In #124 Debezium CDC records are transformed into OpenCDC records.

### CODE REVIEW

File: src/main/java/io/conduit/SourceStream.java [hunk 1]
```diff
@@ -16,127 +16,21 @@
...
-        Logger.get().info("Completed.");
-        stop();
-        responseObserver.onCompleted();
-    }
-
-    private void stop() {
-        Logger.get().info("Stopping...");
-        shouldRun = false;
-    }
+public interface SourceStream extends StreamObserver<Source.Run.Request>, Runnable {
```
Comment (hariso) [inline src/main/java/io/conduit/SourceStream.java:26]: SourceStream was previously a class, not it's an interface. The default implementation is `DefaultSourceStream`.

File: src/main/java/io/conduit/Transformations.java [hunk 2]
```diff
@@ -54,20 +54,13 @@ public static final Record.Builder fromKafkaSource(SourceRecord sourceRecord) {
         return Record.newBuilder()
                 .setKey(getKey(sourceRecord))
                 .setPayload(getPayload(sourceRecord))
-                .setCreatedAt(currentTimestamp());
+                // we need nanoseconds here
+                .putMetadata(OpenCdcMetadata.CREATED_AT, String.valueOf(System.currentTimeMillis() * 1_000_000));
```
Comment (lovromazgon) [inline src/main/java/io/conduit/Transformations.java:58]: Hmm, this sounds like it should be `OpenCdcMetadata.READ_AT`.

File: src/main/proto/opencdc.proto [hunk 3]
```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
```
Comment (lovromazgon) [inline src/main/proto/opencdc.proto:1]: Suggestion: using `buf` could make it easier to manage the proto changes. The java options can be injected in [managed mode](https://docs.buf.build/generate/managed-mode#java). The benefit is you wouldn't have to copy over the proto files in the repo as they can be fetched by buf on demand from the schema registry (https://buf.build/conduitio/conduit-connector-protocol) and updating to a newer version will be easier.

This is out of scope of this PR, it's just a thing I noticed.

File: src/main/java/io/conduit/Transformations.java (hunk 2 above)
Comment (hariso) [reply src/main/java/io/conduit/Transformations.java:58]: Good one. :+1:

**Pull request merged**